ember-app/app/mixins/cards/card-move.js

```diff
@@ -23,13 +23,16 @@ var CardMoveMixin = Ember.Mixin.create({
     moveToTop: function(issue_below){
       var below_order = issue_below._data[this.get("orderKey")]
       var order = below_order / this.orderMultiplier;
+      if(order <= 0){ order = issue_below.get('data.id') * this.minThreshold; }
       while(order < this.minThreshold){ order *= 10; }
 
       return order;
     },
     moveToBottom: function(issue_above){
       var above_order = issue_above._data[this.get("orderKey")];
       var order = above_order * this.orderMultiplier;
+
+      if(order === Infinity){ order = issue_above.get('data.id') / this.maxThreshold; }
```

Comment:
`issue_above.get('data.id') / this.maxThreshold` is going to give us a really small value, not a really big one.

Let's go with `this.maxThreshold / 1e10 * issue_above.get('data.id')`, which will turn `id = 160008709` into `order = 1.60008709e+305`. We'll run into problems if GitHub users create another 9.8 billion issues, which seems like enough breathing room; if you disagree, pick something bigger than 10. :smiling_imp: